error-handlers/error-handler.js

```diff
@@ -1,15 +1,13 @@
 exports.error_handler = (req, res) => {
   res.status(404).send({ msg: "path is not correct" });
 };
-exports.psqlErrors = (error, res) => {
-  console.log("PPSLS");
-  console.log(error);
-  //  if (err.code === "22P02") {
-  //    console.log("PASSED");
-  //    res.status(400).send({ msg: "bad request" });
-  //  }
-};
 
 exports.handle_custom_errors = (err, req, res, next) => {
-  console.log("CUSTOM ERROR");
+  if (err.code === "23503") {
+    res.status(404).send({ msg: "not found" });
+  } else if (err.code === "22P02") {
+    res.status(400).send({ msg: "bad request" });
+  } else {
+    next(err);
+  }
```

Comment:
These are PSQL errors, not custom errors - your custom errors are your promise.rejects()